### PR TITLE
Set up a "Hello Microverse" project

### PR DESCRIPTION
**Pull request by JNC26**
### In this project i did the following:

    I created a new GitHub repo.
    Created a new branch according to the [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) rules.
    Added a .gitignore file.
    Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) and make sure that any files that are not relevant are ignored by git.
    Added a descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
    Added 3 files (each one in a separate commit):
        A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
        A .css file that includes one class with styles for your header (make it in your favorite color).
        A test.md file with the text "This file should be ignored by git".
    Make sure that the last file is actually ignored by git and is not present in your GitHub repository.

### CODE REVIEW

File: MIT.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
+
+###### Please delete this line and the next one
+###### APP TYPE can be a webpage/website, a web app, a software and so on
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the [APP TYPE].
+
+THE [APP TYPE] IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE [APP TYPE] OR THE USE OR OTHER DEALINGS IN THE [APP TYPE].
```
Comment (codecaiine) [inline MIT.md]: - [ ] Good job so far your Licence link works fine :clap: But you forgot to customize it by adding your own name and info. Please replace [YOUR NAME ] with your name to make it more professional :+1:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @JNC26, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Great project implementation :heavy_check_mark: 
- No linters errors :heavy_check_mark: 
- Good commits history :heavy_check_mark: 


## Required Changes ♻️

- [ ] Please make sure that the Pull Request has a descriptive title that gives an idea of what you are doing and what changes are inside. A perfect example will be **Set up a "Hello Microverse" project**  . It will be more professional :+1: 

_Check the comments under the review._

## Optional suggestions

- [ ] Please make sure that in the future your branch name be more descriptive has a feature or action (e.g. 'created_sidebar' or 'added_comments_section') . :smile: 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: MIT.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
```
Comment (codecaiine) [inline MIT.md]: - [ ] [REQUIRED] Good Job so far by implementing what the previous reviewer requested. But please customize your license by  replacing  **[YOUR NAME ]** with your name and own info to make it more professional :smile:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @JNC26 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Great project implementation :heavy_check_mark: 
- No linters errors :heavy_check_mark: 
- Good commits history :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @JNC26 , 

Good vwork implementing the changes requested. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)

## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Descriptive PR👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ugboaja-Uchechi — COMMENTED** [review]
Hi @JNC26 , 

## Status: APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- Great job with your design and functionalities 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**